styles/fonts.css

```diff
@@ -1,48 +1 @@
 /* stylelint-disable max-line-length */
```

Comment:
I believe you won't have to use a font-face rule with typekit but I'm not 100% sure

Reply:
it's been 84 years

Reply:
Yeah, looks like that's what Typekit is going to be giving us. Here's [the kit being used on Milo](https://use.typekit.net/hah7vzn.css), and you can check out the similar thing going on [in the dev console on Spectrum](https://spectrum.adobe.com/).

Reply:
Have you tried including the milo kit for now?